packages/core/src/shared/vscode/setContext.ts

```diff
@@ -14,6 +14,7 @@ type contextKey =
     | 'aws.isDevMode'
     | 'aws.isSageMaker'
     | 'aws.isWebExtHost'
+    | 'aws.isInternalUser'
```

Comment:
I think the key should be named after the feature. The fact that it's currently gated on "internal user" is not something that should be in the key name.